rfcs/text/0000-extend-threat-group-software.md

```diff
@@ -0,0 +1,167 @@
+
+Currently, tactic, technique, and sub-techniques are also included in rules for the Detection Engine, adding software and groups would make for more contextually relevant alerts that could aid in analysis and response operations.
+
+```json5
+{
+// Existing threat fields
+    "threat.framework": "ATT&CK",
+    "threat.tactic.id": "TA0007",
+    "threat.tactic.name": "Discovery",
+    "threat.tactic.reference": "https://attack.mitre.org/tactics/TA0007/",
+    "threat.technique.id": "T1087",
+    "threat.technique.name": "Account Discovery",
+    "threat.technique.reference": "https://attack.mitre.org/techniques/T1087/",
+    "threat.technique.subtechnique.id": "T1087.002",
+    "threat.technique.subtechnique.name": "Domain Account",
+    "threat.technique.subtechnique.reference": "https://attack.mitre.org/techniques/T1087/002/",
+
+// New Software fields
+    "threat.software.id": "S0552",
+    "threat.software.name": "AdFind",
+    "threat.software": {
+      "platforms": [
+        "Windows",
+        "Azure AD"
+      ]
+    },
+    "threat.software.reference": "https://attack.mitre.org/software/S0552/",
+    "threat.software.type": "Tool",
+
+// New Group fields
+    "threat.group.id": "G0037",
+    "threat.group.name": "FIN6",
+    "threat.group.reference": "https://attack.mitre.org/groups/G0037/"
+}
+```
+
+## Source data
+
+<!--
+Stage 1: Provide a high-level description of example sources of data. This does not yet need to be a concrete example of a source document, but instead can simply describe a potential source (e.g. nginx access log). This will ultimately be fleshed out to include literal source examples in a future stage. The goal here is to identify practical sources for these fields in the real world. ~1-3 sentences or unordered list.
+-->
+
+<!--
+Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
+-->
+
+<!--
+Stage 3: Add more real world example source documents so we have at least 2 total, but ideally 3. Format as described in stage 2.
+-->
+
+There are many sources of threat intelligence including open source, closed source, and membership-based ISAC's. Depending on the source the level of details can vary from atomic indicators of compromise (IoC) to higher-order context around threat tactics, infrastructure, and motivations. Generally freely available (open source) intelligence sources will provide details more focused on indicators and commercial intelligence services will provide higher-order details.
```

Comment:
Is this a remnant from https://github.com/elastic/ecs/blob/master/rfcs/text/0008-threat-intel.md#source-data that was left in?

I felt a bit of déjà vu, and then I realized why! 😄

Reply:
Yeah, I can remove this.